docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,194 @@
+
+<Admonition type="info" title="Note">
+
+End-users can attach files such as pdf, video, etc in their issues. For iOS 10 and below, to access files in the “Files” app of iOS, developers will need to add iCloud capability with iCloud Documents services enabled. For more info please refer the [Prerequisites section here.](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/DocumentPickerProgrammingGuide/Introduction/Introduction.html#//apple_ref/doc/uid/TP40014451)
+&nbsp;&nbsp; <Image src="/static/books/ios/icloud-entitlement.png"  width="full" />
+
+</Admonition>
+
+## Initializing Helpshift in your app {#initializing}
+
+1. First, create an app on the Helpshift Dashboard
+
+   <Image src="/static/books/helpshiftx/add_app.png" width="full" />{" "}
+
+2. Create an app with **Android** and **iOS** as selected `Platform`
+   <Image
+     src="/static/books/helpshiftx/create_app_with_android_platform.png"
+     width="full"
+   />{" "}
+
+Helpshift uniquely identifies each registered App with a combination of 2 tokens:
+
+<div className=" compact">
+
+|                          |                                                                                            |
+| ------------------------ | ------------------------------------------------------------------------------------------ |
+| <code>Domain Name</code> | Your Helpshift domain. E.g. _happyapps.helpshift.com_                                      |
+| <code>Platform ID</code> | Your App's unique Platform ID (App's <code>App ID</code> on dashboard is your Platform ID) |
+| <code>Config</code>      | An <code>NSDictionary</code> representing the configuration parameters for the installation. |
+</div>
+
+You can find these by navigating to `Settings`>`SDK (for Developers)` in your agent dashboard.
+Select your App and _check `Android` and `iOS` as a platform_ from the dropdowns and copy
+the 2 tokens to be passed when initializing Helpshift.
+
+<Image src="/static/books/helpshiftx/copy_install_keys.png"  width="full" />
+
+You should now import `install` function from `helpshift-plugin-sdkx-react-native`. Copy the `domain` and `platformID` from the dashboard and use in `install` function.
+
+
+Call the `install` function before any other interaction with the Helpshift SDK in `App.tsx`.(preferably in the useEffect() callback in react native call)
+
+The `install` call takes 3 parameters - App ID, Domain and config. App ID and domain are mandatory, while config is optional, which means you can pass an empty map. The meaning of different keys and their possible values for Android and iOS can be found [here](/sdkx-react-native/sdk-configuration)
+
+### Initialize
+```
+// Import the methods required for your integration.
+// You can refer the index.tsx file to see the methods available
+// to import. In this example, we are not importing all the
+// methods.
+
+import {
```

Comment:
// this code would do 
import {
  install
} from 'helpshift-plugin-sdkx-react-native';

Reply:
updated.